src/commands/config/starboard/blacklist/remove.ts

```diff
@@ -0,0 +1,75 @@
+import config from "adapters/config"
+import { Message } from "discord.js"
+import { APIError, GuildIdNotFoundError } from "errors"
+import { BlacklistChannelRepostConfigRequest, Command } from "types/common"
+import { getCommandArguments, parseDiscordToken } from "utils/commands"
+import { PREFIX } from "utils/constants"
+import { composeEmbedMessage, getErrorEmbed } from "utils/discordEmbed"
+
+const command: Command = {
+  id: "starboard_blacklist_remove",
+  command: "remove",
+  brief: "Remove channel from blacklist cannot bookmark",
+  category: "Config",
+  onlyAdministrator: true,
+  run: async function (msg: Message) {
+    if (!msg.guild) {
+      throw new GuildIdNotFoundError({ message: msg })
+    }
+    const args = getCommandArguments(msg)
+    if (args.length === 3) {
+      return { messageOptions: await this.getHelpMessage(msg) }
+    }
+    const channelArg = args[3]
+    const { isChannel, value } = parseDiscordToken(channelArg)
+    const channel = await msg.guild?.channels.fetch(value)
+    if (!isChannel || !channel) {
+      return {
+        messageOptions: {
+          embeds: [
+            getErrorEmbed({
```

Comment:
is this an expected error? note that if you don't throw error we won't be able to log failed commands